fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/PhysicalDataModel.java

```diff
@@ -339,24 +339,26 @@ public void visit(DataModelVisitor v, final String tagGroup, final String tag, S
         // visit just the matching subset of objects. If a transactionSupplier has been provided, we break up the
         // operation into multiple transactions to avoid transaction size limitations (e.g. with Citus FK creation)
         if (transactionSupplier != null) {
+
+            
             ITransaction tx = transactionSupplier.get();
             try {
                 int count = 0;
                 for (IDatabaseObject obj: allObjects) {
                     if (tag == null || obj.getTags().get(tagGroup) != null && tag.equals(obj.getTags().get(tagGroup))) {
-                        if (++count == 10) {
-                            // commit the current transaction and start a fresh one
-                            tx.close();
-                            tx = transactionSupplier.get();
-                            count = 0;
-                        }
-
                         try {
+                            logger.info("Visiting[" + count + "] " + obj.getName());
```

Comment:
are you sure we want this at info level?  schema creation is already quite verbose

Reply:
Yeah, cruft from debugging. Changed to fine with a guard.